test/parallel/test-crypto-authenticated.js

```diff
@@ -212,7 +212,7 @@ for (const test of TEST_CASES) {
     cipher.update('01234567', 'hex');
     cipher.final();
     const tag = cipher.getAuthTag();
-    assert.strictEqual(tag.toString('hex'), fullTag.substr(0, 2 * e));
+    assert.strictEqual(tag.toString('hex'), fullTag.slice(0, 2 * e));
```

Comment:
I think this (and similar changes) should land as its own PR. Let me know if you need help

Reply:
I think it's fine here but feel free to create a separate PR if you want